tests/test_models/test_editors/test_stable_diffusion/test_stable_diffusion.py

```diff
@@ -84,7 +84,7 @@ def __call__(self, x, attention_mask):
 
 
 model = dict(
-    type='StableDiffusion',
+    type='StableDiffusionXL',
```

Comment:
This file is for testing stablediffusion, and this line should not be changed. Maybe a mistake.